### ISSUE TITLE
Generate from `git log`

### ISSUE THREAD

**Issue by aarondfrancis** [deuill/grawkit#8]
These graphs are so beautiful, but I can't figure out how to generate one from the output of `git log`. Do you have any ideas or directions you could point me in?

Thanks!

**Comment by deuill** [comment]
I don't believe there's a single `git` command that allows for, essentially, reverse-engineering the commands used for generating the graph, nor does `grawkit` support reading from a more machine-readable format, as it's generally intended for short demonstrations, not full visualizations of repositories.

That said, there might be some way of extracting the commands from `git reflog` or `git ref-list` and a bit of shell-fu, as all the information needed *is* available, it seems, just not in the right format.
